packages/components/psammead-section-label/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 0.1.1 | [PR#610](https://github.com/bbc/psammead/pull/610) Add visuallyHidden prop. |
```

Comment:
It looks like you need to bump it in package.json and package lock, i guess it got lost in the latest pull